packages/binding-http/test/http-server-oauth-tests.ts

```diff
@@ -79,7 +79,7 @@ class OAuthServerTests {
 
     @test async "should configure oauth"() {
         /* eslint-disable dot-notation */
-        this.server["httpSecurityScheme"].should.be.equal("OAuth");
+        this.server["supportedSecuritySchemes"].should.contain("oauth2");
         expect(this.server["oAuthValidator"]).to.be.instanceOf(EndpointValidator);
```

Comment:
Minor: since we are there we could resolve some of the eslint warnings and use the dot notation..

Reply:
No, we can't because it is a private member.